src/views/product-landing/server.ts

```diff
@@ -1,100 +1,100 @@
-        // TODO: behavior of static default to "neutral"). Would eliminate
-        // TODO: the need to add iconBrandColor to cards (unless doing so
-        // TODO: as an explicit override).
-        // ensure cards with icons have an iconBrandColor,
-        // falling back to the current product if not set
-        /* eslint-disable-next-line no-case-declarations */
-        let defaultIconColor
-        if (product.slug === 'hcp' || product.slug === 'sentinel') {
-          defaultIconColor = 'neutral'
-        } else {
-          defaultIconColor = product.slug
-        }
-        return {
-          ...block,
-          cards: block.cards.map((card) => ({
-            ...card,
-            iconBrandColor: card.iconBrandColor || defaultIconColor,
-          })),
-        }
-      default:
-        return block
-    }
-  })
-
-  const navData = [
-    ...product.sidebar.landingPageNavData,
-    { divider: true },
-    ...product.sidebar.resourcesNavData,
+  /**
+   * TODO: content is raw placeholder for now
+   * - Need to add "headings" that aren't placeholders.
+   *   These will be derived from content blocks
+   *   (mainly heading blocks, but also others, based on design intent)
+   * - Need to augment things like tutorialSlugs to fill in that data
+   *   Will close: https://app.asana.com/0/1201010428539925/1201654639085764/f
+   * - Need to build out the client-side components, whose props APIs will
+   *   inform what content structure we'll want to return for use on the client.
+   *   Intent being to return the minimum data necessary to render
+   *   all content block components on the page.
+   */
+  const placeholderHeadings = [
+    {
+      title: `What is ${product.name}`,
+      slug: 'placeholder-slug-1',
+      level: 2,
+    },
+    {
+      title: 'Getting Started',
+      slug: 'placeholder-slug-2',
+      level: 2,
+    },
+    {
+      title: 'Use Cases',
+      slug: 'placeholder-slug-3',
+      level: 2,
+    },
+    {
+      title: 'Case Studies',
+      slug: 'placeholder-slug-4',
+      level: 2,
+    },
   ]
 
+  /**
+   * Gather up our static props package & return it
+   */
   return {
     content: CONTENT,
     layoutProps: {
-      headings: CONTENT.blocks
-        .filter((s) => s.type == 'heading')
-        .map(({ heading, slug, level }) => ({
-          title: heading,
-          slug: slug,
-          level,
-        })),
+      headings: placeholderHeadings,
       breadcrumbLinks: [
         { title: 'Developer', url: '/' },
         { title: product.name, url: `/${product.slug}` },
       ],
       sidebarProps: {
-        menuItems: navData,
+        /**
+         * TODO: we may need to map MenuItem entries to
+         * EnrichedNavItem entries? For now, I've casted them
+         * to sidestep the issue, but this is likely not
+         * a good long-term solution. Related task:
+         * https://app.asana.com/0/1202022787106807/1201602267333015/f
```

Comment:
I might be typing this wrong? Or this may be related to our task to break up the `MenuItem` interface? I think this wasn't an issue previously because the return type was `$TSFixMe`.

Intent is to resolve this `TODO` more thoroughly in a subsequent PR, for now I wanted to at least have a clear return type set up for `generateStaticProps`.